packages/gatsby-plugin-sharp/package.json

```diff
@@ -11,6 +11,7 @@
     "async": "^2.1.2",
     "bluebird": "^3.5.0",
     "fs-extra": "^7.0.0",
+    "got": "^9.6.0",
```

Comment:
Only potential issue here is got needs node version >= 8.6 ( https://github.com/sindresorhus/got/blob/v9.6.0/package.json#L8-L10 ) while we technically should work on >= 8.0

thoughts on this?

Reply:
Wow good catch, I can revert to something like axios or request

Reply:
Or use `got@7` or `got@8` which works up to node 4 ;)